metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -25,6 +25,7 @@ class KafkaConnectSourceConfig(ConfigModel):
     password: Optional[str] = None
     cluster_name: Optional[str] = "connect-cluster"
     env: str = builder.DEFAULT_ENV
+    construct_lineage_workunits: bool = True
```

Comment:
since we are starting to see this kind of config show up in other sources as well.. 
   e.g. sql connectors have : `enable_sql_profiling: True` ... etc... thinking it would be good to standardize the on-off switches for extracting the sub-types of metadata across all sources. 

  Any suggestions for a convention we could follow?